### PR TITLE
Use yarn instead of npm

### PR DESCRIPTION
**Pull request by mitsuhiko**
@benvinegar

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,14 +9,15 @@ install-python:
 	pip install ujson
 	pip install "file://`pwd`#egg=sentry[dev]"
 
-install-npm:
+install-yarn:
 	@echo "--> Installing Node dependencies"
-	npm install
+	@hash yarn 2> /dev/null || npm install yarn
```
Comment (mattrobenolt) [inline Makefile]: Doesn't this need `npm install -g yarn` ?

Comment (benvinegar) [inline Makefile]: @mitsuhiko ^

**Comment by benvinegar** [comment]
https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn

**Comment by benvinegar** [comment]
Just the `dist` Travis task is failing now: https://travis-ci.org/getsentry/sentry/jobs/190147193

**Comment by benvinegar** [comment]
Woot, Travis is green.

**Comment by mitsuhiko** [comment]
Looking good but since it's my PR i can't sign this off.

**Review by benvinegar — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "moment": "2.10.6",
     "moment-timezone": "0.4.1",
     "node-libs-browser": "0.5.3",
+    "node-zopfli": "2.0.2",
```
Comment (mattrobenolt) [inline package.json]: Remove this. We don't use zopfli anymore.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**